### PR TITLE
More updates and changes...

### PR DESCRIPTION
**Pull request by Talamar1**

### CODE REVIEW

**Comment by mcmonkey4eva** [comment]
When I say "Don't do `<x>`", how much crack do you have to be smoking to just do it anyway?

**Comment by Talamar1** [comment]
I was bored and you were slow....sometimes you forget to do things and I already had this done and someone using it.

File: src/main/java/net/aufdemrand/denizen/events/entity/EntityDamagedScriptEvent.java [hunk 1]
```diff
@@ -179,6 +195,10 @@ public void onEntityDamaged(EntityDamageEvent event) {
         damage = new Element(event.getDamage());
         final_damage = new Element(event.getFinalDamage());
         cause = new Element(event.getCause().name().toLowerCase());
+        iih = null;
+        if (entity.isLivingEntity()) {
+            iih = new dItem(entity.getLivingEntity().getEquipment().getItemInHand());
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/events/entity/EntityDamagedScriptEvent.java]: YOU DIDN'T EVEN TAKE THE ITEM FROM THE RIGHT ENTITY D:<

**Comment by Talamar1** [comment]
I fixed it <.<
Also, I should clean up the way I handle the damage it handled, keeping it as Element requires multiple calls for conversion.  I should keep it a double until it is needed in Context like I do in other scripts.

**Pull request closed without merging**